pkg/cmd/issue/view/fixtures/issueView_previewWithEmptyBody.json

```diff
@@ -7,7 +7,7 @@
         "body": "",
         "title": "ix of coins",
         "state": "OPEN",
-        "created_at": "2011-01-26T19:01:12Z",
+        "createdAt": "2011-01-26T19:01:12Z",
```

Comment:
I fixed up a bunch of these that were being parsed incorrectly and leading to the `292 years` strings in the tests.